lib/sr.py

```diff
@@ -166,7 +167,7 @@ def get_type(self) -> str:
             self._type = self.pool.master.xe("sr-param-get", {"uuid": self.uuid, "param-name": "type"})
         return self._type
 
-    def create_vdi(self, name_label: str, virtual_size: int = 64, image_format: Optional[str] = None) -> VDI:
+    def create_vdi(self, name_label: str, virtual_size: int = 1 * GiB, image_format: Optional[str] = None) -> VDI:
```

Comment:
That feels strange at first look, but we were indeed requesting a default VDI size of *64 bytes*, wow.  Maybe mention this in commit message to avoid too many readers taking time to double-check that new value would really do what it's meant to.